packages/components/src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import * as React from 'react';
+
+import { cn } from '~/lib/utils';
+
+const Tabs = TabsPrimitive.Root;
+
+const TabsList = React.forwardRef<
+  React.ElementRef<typeof TabsPrimitive.List>,
+  React.ComponentPropsWithoutRef<typeof TabsPrimitive.List>
+>(({ className, ...props }, ref) => (
+  <TabsPrimitive.List
+    className={cn('mb-8 flex list-none items-start overflow-x-auto text-base', className)}
+    ref={ref}
+    {...props}
+  />
+));
+
+TabsList.displayName = 'TabsList';
```

Comment:
I prefer if we get the `displayName` from the radix primitive instead, like we do in other components.

```suggestion
TabsList.displayName = TabsPrimitive.List.displayName;
```

Reply:
I've also changed for `TabsTrigger` & `TabsContent`.